cassiopeia/data/CassiopeiaTree.py

```diff
@@ -1276,6 +1475,7 @@ def compute_dissimilarity_map(
             dissimilarity_function,
             weights,
             self.missing_state_indicator,
+            numbaize=numbaize
```

Comment:
Unfortunately, I've spent way too much time trying to get it to numbaize with no progress. Numba doesn't play well with variable length anything, and since each tuple may have variable number of states this isn't easily numbaizable :(